db/migrate/20211203210730_change_kind_from_products.rb

```diff
@@ -0,0 +1,16 @@
+class ChangeKindFromProducts < ActiveRecord::Migration[6.1]
+  def change
+    reversible do |dir|
+      dir.up do
+        Product.where(kind: 'Food').update_all(kind: 0)
+        Product.where(kind: 'Drink').update_all(kind: 1)
+        Product.where(kind: 'Combo').update_all(kind: 2)
+        change_column :products, :kind, :integer, default: 0
```

Comment:
let's discuss

Reply:
Sir, migration fixed.
Added new column, updated values to that and then renamed.